libraries/Filter/NotchFilter.cpp

```diff
@@ -38,23 +38,6 @@ void NotchFilter<T>::calculate_A_and_Q(float center_freq_hz, float bandwidth_hz,
     }
 }
 
-/*
-  initialise filter
- */
-template <class T>
-void NotchFilter<T>::init(float sample_freq_hz, float center_freq_hz, float bandwidth_hz, float attenuation_dB)
```

Comment:
This is about to be used by https://github.com/ArduPilot/ardupilot/pull/24411

Reply:
Yes please don't remove this!

Reply:
It's also used by blimb (I had no idea it was being used like that). I've put it back in.